src/subject_name/mod.rs

```diff
@@ -10,20 +10,23 @@
 // ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
 // WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
 // ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
-// OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+// OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.;
 
 mod dns_name;
 pub use dns_name::{DnsNameRef, InvalidDnsNameError};
 
-/// Requires the `alloc` feature.
+// Requires the `alloc` feature.
 #[cfg(feature = "alloc")]
 pub use dns_name::DnsName;
 
-#[allow(clippy::module_inception)]
 mod name;
 pub use name::{InvalidSubjectNameError, SubjectNameRef};
 
-pub mod ip_address;
+mod ip_address;
```

Comment:
I think is weird that the `name` module is not public but `ip_address` is. In this case I think it would be good to have them the same for consistency.

I can update if you prefer to make `name` public too instead.

Reply:
Okay, makes sense! I don't think we're exporting enough items for making a modular structure public, so I think this change is an improvement.